examples_lib/reading_timestamp.go

```diff
@@ -241,15 +241,17 @@ func RunTimestampArray() {
 	// Write data and metadata
 	t1 := getTimestamp()
 	writeTimestampArray(tmpDir1, "meta_key", "Write1", t1, 0)
-	time.Sleep(2000 * time.Millisecond)
+	// Wait a few milliseconds to ensure that we are not going to overwrite
+	// data.
```

Comment:
instead of "we are not going to overwrite the data" (which we want to do), it might be better to say "our writes are in different milliseconds to ensure correct ordering".